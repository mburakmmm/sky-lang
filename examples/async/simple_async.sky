# Simple async/await test

async function fetchData(): int
  print("Fetching data...")
  return 42
end

async function main
  print("Start")
  let result = await fetchData()
  print("Result:")
  print(result)
  print("Done")
end

