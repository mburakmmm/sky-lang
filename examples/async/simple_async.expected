Start
Fetching data...
Result:
42
Done

